src/python/README.md

```diff
@@ -0,0 +1,53 @@
+# Python Scripts for Merge Conflict Analysis
+
+
```

Comment:
There is a lot of extraneous whitespace in this README file.

Reply:
I believe this is standard. I think is designed to be seen from an end-users perspective like in the browser it looks normal.

https://github.com/cactusbranch01/AST-Merging-Ben-Analysis/blob/ben-main/src/python/README.md

If you think I should change this, though, I would be happy to.